src/main/java/cubicchunks/lighting/LightingManager.java

```diff
@@ -46,7 +46,8 @@ public LightingManager(ICubicWorld world) {
 
 		this.skylightCubeDiffuseProcessor = new SkyLightCubeDiffuseProcessor(world, "Sky Light Diffuse", 5);
 
-		this.firstLightProcessor = new FirstLightProcessor("First Light", world.getCubeCache(), 1);
+		// TODO: Setting GeneratorStage to null. Analyze effects.
+		this.firstLightProcessor = new FirstLightProcessor(null, "First Light", world.getCubeCache(), 1);
```

Comment:
LightingManager is used on both the client and the server. Because the client does not have a copy of the GeneratorPipeline it is not aware of the stages.